examples/hacker-news/app/app.re

```diff
@@ -1,15 +1,70 @@
 open Brisk_macos;
 
+
 let () = {
   open Cocoa;
 
-  let appName = "BriskMac";
-
   Application.init();
 
   Application.willFinishLaunching(() => {
-    let menu = UiMenu.makeMainMenu(appName);
-    Menu.add(~kind=Main, menu);
+    open Brisk_macos.Menu;
```

Comment:
I badly want to keep entry point as thin as possible to make it easy for newbies to read it, but instead it's getting bloated.

Reply:
I'll put the menu under `MainMenu.menu` value.